### DISCUSSION TITLE
Unable to edit ACLs for file in S3

### DISCUSSION THREAD

**Discussion by elmercado07** [iterate-ch/cyberduck#15093] [Q&A]
Ever since upgrading to MacOS Sanoma (Public Beta) I have been unable to assign permissions to uploaded files in S3 using this product. I can upload just fine and access the files Permissions via Info. But when I got to assign new permissions, I select who I want to share permissions with (Everyone), and go to assign Read rights, and nothing happens. The drop-down disappears, and nothing saves.

Please assist, as this is causing a lot of issues. 
![Cyberduck issue](https://github.com/iterate-ch/cyberduck/assets/34006293/236c596e-5d4a-4f67-b403-5639cb3eecfc)

I'm not sure if it's because I am on the beta version of MacOS, but that is when I noticed the problem started happening.

**Comment by dkocher** [comment]
I cannot reproduce the problem running on macOS Sonoma 14.0 Beta.
